pontoon/base/static/js/main.js

```diff
@@ -597,7 +597,7 @@ $(function() {
     Pontoon.startLoader();
 
     $.ajax({
-      url: '/api/v1/user/' + Pontoon.user.email + '/',
+      url: '/api/v1/user/' + $('#server').data('email') + '/',
```

Comment:
Maybe we should move initialization of user data before this line to keep one source of data for the user profile in javascript?